djoser/views.py

```diff
@@ -130,11 +130,12 @@ def _action(self, serializer):
     def get_users(self, email):
         if self._users is None:
             email_field_name = get_user_email_field_name(User)
-            active_users_kwargs = {
-                email_field_name + '__iexact': email, 'is_active': True
+            filter_users_kwargs = {
+                email_field_name + '__iexact': email
             }
```

Comment:
Since these kwargs are rather short now, you might as well simplify the `users` into:
```
users = User._default_manager.filter(**{email_field_name + '__iexact': email})
```
with a line break if necessary.

That way, it's still readable with less lines to read.